rttr-ru.po

```diff
@@ -1138,11 +1145,10 @@ msgid "Don't forget to check the addon configuration!"
 msgstr "Не забудьте проверить конфигурацию дополнений!"
 
 #: libs/s25main/desktops/dskOptions.cpp:175
-msgid ""
+-msgid ""
```

Comment:
this one is weird

Reply:
Right, looks like a broken conflict resolution. Maybe: 
```suggestion
msgid ""
```

Reply:
Oops, yes... Technical error (
Is it necessary to commit and pull request again?

Reply:
Fixed a few more bugs, including forgetting quotes and extra spaces... Prode should be fine

Reply:
>  Is it necessary to commit and pull request again?

As you have likely seen adding the commits to the branch of this PR is enough. Thanks for fixing this!